scala/scala_toolchain.bzl

```diff
@@ -137,6 +138,10 @@ scala_toolchain = rule(
             default = True,
             doc = "Enable writing of statsfile",
         ),
+        "use_argument_file_in_runner_for_improved_performance": attr.bool(
```

Comment:
I think shorter name is better: "use_argument_file_in_runner". Reasoning is already described in doc.

Reply:
👍🏽 will change